libvast/src/store.cpp

```diff
@@ -400,7 +400,9 @@ system::default_active_store_actor::behavior_type default_active_store(
             self->quit(std::move(error));
         },
         [self](stream_state& stream_state, const caf::error& error) {
-          if (error) {
+          if (ec::end_of_input == error) {
+            VAST_WARN("Encountered end of input before shutdown");
+          } else if (error) {
```

Comment:
What's the context of this change?

Please also rewrite the log message in the usual style, e.g.,

> store encountered unexpected end of input before shutdown

Reply:
This is the issue with our source streams propagating ec::end_of_input error when the input ended.
Previously this error couldn't propagate here for some reason. Now it is sometimes visible and sometimes not. I believe the error was only introduced to signalize the end of input so we kind of ignore it instead of treating it like a real error

Reply:
Okay, that makes sense. We need to re-evaluate the stream on import with pipelines anyways, so I think we can solve this sometime in the future rather than now. Please just rephrase the error message then.